docs/dev-setup.md

```diff
@@ -0,0 +1,86 @@
+# Dev Setup
+
+## Requirements
+
+ODH requires the following to run:
+
+- [NodeJS and NPM](https://nodejs.org/)
+  - Node recommended version -> `18.16.0`
+  - NPM recommended version -> `9.6.7`
+- [OpenShift CLI](https://docs.openshift.com/container-platform/4.12/cli_reference/openshift_cli/getting-started-cli.html)
+- [kustomize](https://github.com/kubernetes-sigs/kustomize)
+- [podman](https://github.com/containers/podman)
+- Have access to [Quay.io](https://quay.io/)
+
+### Additional tooling requirements
+
+- [OpenShift CLI, the "oc" command](https://docs.openshift.com/container-platform/4.12/cli_reference/openshift_cli/getting-started-cli.html)
+- [Podman](https://github.com/containers/podman)
+- [Quay.io](https://quay.io/)
+
+## Development
+
+1. Clone the repository
+
+      ``` bash
+      git clone https://github.com/opendatahub-io/odh-dashboard
+      ```
+
+  2. Within the repo context, install project dependencies
+
+     ```bash
+     cd odh-dashboard && npm install
+     ```
+
+### Build project
+
+  ```bash
+  npm run build
+  ```
+
+### Serve development content
+
+This is the default context for running a local UI.  Make sure you build the project using the instructions above prior to running the command below.
+
+  ```bash
+  npm run start
+  ```
+
+For in-depth local run guidance review the [contribution guidelines](./CONTRIBUTING.md#Serving%20Content).
+
+### Testing
+
+Run the tests.
+
+  ```bash
+  npm run test
+  ```
+
+Run storybook a11y tests and interaction tests.
+
+  ```bash
+  npm run storybook
+  npm run test:storybook
```

Comment:
We neither have a  script for storybook nor test:storybook

Reply:
@Gkrumbach07 @christianvogt I think when you guys changed the landscape you didn't update the docs.

Could you guys spin that off in another issue?

Reply:
I think it was updated in a different doc but not this one

Reply:
here is where the up to date details are: https://github.com/opendatahub-io/odh-dashboard/blob/main/CONTRIBUTING.md#debugging-and-testing

im guessing you got that text from the readme

Reply:
`dev-setup.md` is entirely what was on the root README, so the mistake exists on `main` today